proto/tendermint/liquidity/v1beta1/liquidity.proto

```diff
@@ -124,15 +128,15 @@ message Params {
             format: "sdk.Dec"
         }];
 
-    // The smallest unit batch size for every liquidity pool
-    uint32 unit_batch_size = 9 [
-        (gogoproto.moretags) = "yaml:\"unit_batch_size\"",
+    // The smallest unit batch height for every liquidity pool
+    uint32 unit_batch_height = 9 [
+        (gogoproto.moretags) = "yaml:\"unit_batch_height\"",
         (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
             example: "\"1\"",
             format: "uint32"
         }];
 
-    //    // fees paid for each batch messages
+    //    // fees paid for each batch messages, to be added v2
     //    repeated cosmos.base.v1beta1.Coin liquidity_msg_fee = 5 [
     //        (gogoproto.moretags)   = "yaml:\"liquidity_msg_fee\"",
     //        (gogoproto.nullable)   = false,
```

Comment:
```suggestion
    //    // fees paid for each batch messages, to be added v2
    //    repeated cosmos.base.v1beta1.Coin liquidity_msg_fee = 5 [
    //        (gogoproto.moretags)   = "yaml:\"liquidity_msg_fee\"",
    //        (gogoproto.nullable)   = false,
```

Reply:
hmmm best practice is to avoid mention of future features, can we comment this code and comment out? 
see https://developers.google.com/style/future